libcontainer/cgroups/cgroups.go

```diff
@@ -48,4 +48,7 @@ type Manager interface {
 
 	// Whether the cgroup path exists or not
 	Exists() bool
+
+	// OOMKill reports OOM kill count for the cgroup.
+	OOMKill() (uint64, error)
```

Comment:
Maybe rename this to `OOMKillCount()` or similar?

Reply:
Right, otherwise it looks like an action :)

Reply:
fixed